articles/spring-cloud/quickstart.md

```diff
@@ -180,15 +180,6 @@ In Visual Studio, create an ASP.NET Core Web application named as "hello-world"
    }
    ```
 
-   ```csharp
-   public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
```

Comment:
Please also remember to remove the `Configure` method in the description (Line 168).